packages/mdjs/src/mdjsProcess.js

```diff
@@ -9,53 +9,65 @@ const raw = require('rehype-raw');
 const htmlStringify = require('rehype-stringify');
 const htmlSlug = require('rehype-slug');
 const htmlHeading = require('rehype-autolink-headings');
+const rehypePrism = require('rehype-prism-template');
 
 const { mdjsParse } = require('./mdjsParse.js');
 const { mdjsStoryParse } = require('./mdjsStoryParse.js');
 
 async function mdjsProcess(
   mdjs,
-  { rootNodeQueryCode = 'document', mdjsStoryParseOptions = {} } = {},
+  { rootNodeQueryCode = 'document', mdjsStoryParseOptions = {}, htmlHeadingOptions = {} } = {},
 ) {
   const parser = unified()
     .use(markdown)
     .use(mdjsParse)
     .use(mdjsStoryParse, mdjsStoryParseOptions)
     .use(remark2rehype, { allowDangerousHTML: true })
+    .use(rehypePrism)
     .use(raw)
     .use(htmlSlug)
-    .use(htmlHeading)
+    .use(htmlHeading, htmlHeadingOptions)
     .use(htmlStringify);
 
+  // const defaultPlugins = [{ plugin: markdown, options:  }, remark2rehype]
```

Comment:
Do we wanna keep this around? I think we'll need a plugins interface when we go "full public" on this as a plugin...but likely that comes with a more concerted effort later.